lte/gateway/c/sctpd/sctpd_downlink_impl.cpp

```diff
@@ -19,13 +19,15 @@
 #include <unistd.h>
 #include "sctpd.h"
 #include "util.h"
-
+#include "magma_logging.h"
 namespace magma {
 namespace sctpd {
 
+ using namespace std;
```

Comment:
It is better to use std:: prefix to give explicit indication on the objects used from std namespace.